src/test/test_address.c

```diff
@@ -221,7 +221,7 @@ test_address_ifaddrs_to_smartlist(void *arg)
    ifa_ipv6->ifa_dstaddr = NULL;
    ifa_ipv6->ifa_data = NULL;
 
-   smartlist = ifaddrs_to_smartlist(ifa, AF_UNSPEC);
+   smartlist = ifaddrs_to_smartlist(ifa, AF_UNSPEC, 0);
```

Comment:
For every test that you added a "0" (non-loopback), please write a version of the test that tests "1" (loopback).